modules/system-management/pages/management-commands.adoc

```diff
@@ -0,0 +1,800 @@
+= Command glossary
+:description: List of commands for system management
+:page-aliases: gadmin:management-commands.adoc
```

Comment:
I don't think this is the right alias? It should be the previous name of the page. I think this would be management-with-gadmin.adoc?